apps/keyboard/js/settings/general_panel.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+/* global PanelBase, ViewBase, GeneralSettingsGroupView,
+          HandwritingSettingsGroupView */
+
+(function(exports) {
+
+var GeneralPanel = function(app) {
+  this._menuUDItem = null;
+
+  this.app = app;
+};
+
+GeneralPanel.prototype = Object.create(PanelBase.prototype);
+
+GeneralPanel.prototype.CONTAINER_ID = 'general';
+GeneralPanel.prototype.USER_DICT_ITEM_ID = 'menu-userdict';
+
+GeneralPanel.prototype.start = function() {
+  PanelBase.prototype.start.call(this);
+
+  this._menuUDItem = document.getElementById(this.USER_DICT_ITEM_ID);
+
+  this.views.general = new GeneralSettingsGroupView(this.app);
```

Comment:
I think you are saving the views in the global `PanelBase#views` instead of the `views` object for this instance. Is it expected?